README.md

```diff
@@ -1,38 +1,36 @@
 # broadcaster
 ![img](https://img.shields.io/github/go-mod/go-version/Decentr-net/go-broadcaster) ![img](https://img.shields.io/github/v/tag/Decentr-net/go-broadcaster?label=version)
 
-Package which simply broadcasting messages to cosmos based blockchain node
+Package which simplifies broadcasting messages to decentr blockchain node
 
 ## Example
 
 ```
 import (
+    . "github.com/Decentr-net/decentr/testutil"
```

Comment:
what for?

Reply:
it's ready-to-go example . so we need some address to follow